packages/path_provider/path_provider/example/linux/.gitignore

```diff
@@ -0,0 +1 @@
+flutter/ephemeral
```

Comment:
linux directory in a non-linux plugin.

Reply:
`path_provider` supports Linux; this isn't a non-Linux plugin.